womtool/src/main/scala/womtool/cmdline/WomtoolCommandLineParser.scala

```diff
@@ -82,7 +82,7 @@ class WomtoolCommandLineParser extends scopt.OptionParser[PartialWomtoolCommandL
 
   cmd("upgrade")
     .action((_, c) => c.copy(command = Option(Upgrade)))
-    .text("(Experimental) Automatically upgrade the WDL to the latest stable version and output the result." + System.lineSeparator)
+    .text("Automatically upgrade the WDL to the latest stable version and output the result." + System.lineSeparator)
```

Comment:
Maybe `"stable"`=>`"released WDL"`

Reply:
Come to think of it, this feels like it might become a lie in the future (ie as soon as WDL 1.1 is released)... 

I think it may behoove us to be explicit here: "this will upgrade WDL draft-2 to WDL 1.0"